tictactoe.py

```diff
@@ -31,17 +32,21 @@
 	"bg.png": "https://raw.githubusercontent.com/Tashar02/tic-tac-toe/main/bg.png",
 }
 
-for filename, url in assets.items():
+# Downloads a file if it does not exist
+def download_asset(filename, url):
 	if not os.path.exists(filename):
 		try:
 			print(f"\nDownloading {filename}...")
-			wget.download(url, filename)
+			request.urlretrieve(url, filename)
 		except Exception as e:
 			print(f"Error downloading {filename}: {e}")
 			print("Exiting program!")
 			pg.quit()
 			sys.exit()
 
+for filename, url in assets.items():
```

Comment:
Why not put the "for" loop inside the download_asset()?

Reply:
Good call,  The `for` loop is now inside the download_assets() function in the updated PR.

Reply:
Thanks, please check the comments on your latest commit "[tictactoe: Refactor asset download logic](https://github.com/Tashar02/tic-tac-toe/pull/1/commits/5dc7fd76fab24df6a74f757cc31f1d8eb8f1fc6d)" I made and do the changes and squash everything since it's not worth making 2-3 patches for such a silly thing.